gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -439,6 +439,39 @@ def create_revel_grch38_ht() -> hl.Table:
     return final_ht
 
 
+def create_phylop_grch38_ht() -> hl.Table:
+    """
+    Convert PhyloP scores to Hail Table.
+
+    .. note::
+    BigWig format of Phylop was download from here:
+    https://cgl.gi.ucsc.edu/data/cactus/241-mammalian-2020v2-hub/Homo_sapiens/241-mammalian-2020v2.bigWig
+    and converted it to bedGraph format with bigWigToBedGraph from the kent packages
+    of UCSC (https://hgdownload.cse.ucsc.edu/admin/exe/) with the following command:
+    `./bigWigToBedGraph ~/Downloads/241-mammalian-2020v2.bigWig ~/Downloads/241-mammalian-2020v2.bedGraph`
+    The bedGraph file is bigzipped before importing to Hail.
+    Different to other in silico predictors, Phylop HT is not keyed by locus
+    & alleles, we have a score per base pair, so we won't have any score for deletions.
+
+    :return: Hail Table with Phylop Scores for GRCh38
+    """
+    bg_path = "gs://gnomad-insilico/phylop/Human-GRCh38-Phylop-241-mammalian-2020v2.bedGraph.bgz"
+    columns = ["chr", "start", "end", "phylop"]
+    ht = hl.import_table(
+        bg_path,
+        min_partitions=1000,
+        impute=True,
+        no_header=True,
+    ).rename({f"f{i}": c for i, c in enumerate(columns)})
+
+    ht = ht.annotate(pos=hl.range(ht.start + 1, ht.end + 1))
+    ht = ht.explode("pos")
+    ht = ht.annotate(locus=hl.locus(ht.chr, ht.pos, reference_genome="GRCh38"))
+    ht = ht.select("locus", "phylop")
```

Comment:
This whole block code be 
```suggestion
    ht = hl.import_bed(bg_path, min_partitions=1000)
    ht = ht.rename({"target":"phylop"})
```
This will key by interval. Since we only have locus anyways, we will need to join by ht.locus, not ht.key. Hail will handle the logic of if a locus is in an interval so we can just keep the file as intervals. This is from [their docs on import_bed](https://hail.is/docs/0.2/methods/impex.html#hail.methods.import_bed).
```
bed = hl.import_bed('data/file2.bed')
result = dataset.annotate_rows(cnv_id = bed[dataset.locus].target)
```

Reply:
Qin shared a slack thread with me where KC and her decided to explode the intervals.